### ISSUE TITLE
version info not in releases

### ISSUE THREAD

**Issue by graphaelli** [elastic/apm-server#479]
#404 started printing version information on apm-server startup.  That relies on build time flags that do not appear to be set in releases.  For 6.2, startup will look like:

```
2018/01/11 14:55:28.663632 beat.go:203: INFO Setup Beat: apm-server; Version: 6.2.0
2018/01/11 14:55:28.666212 cfgwarn.go:11: WARN BETA: You are using a beta release of the APM Server.
2018/01/11 14:55:28.667101 server.go:28: INFO Starting apm-server [unknown built unknown]. Hit CTRL-C to stop it.
```

`apm-server version` reports:
```
apm-server version 6.2.0 (amd64), libbeat 6.2.0
```

so that's good.

Things get a bit worse on `master`.  The startup message still says unknown but now `apm-server version` reports:
```
apm-server version 7.0.0-alpha1 (amd64), libbeat 7.0.0-alpha1 [unknown built unknown]
```

This issue affects beats as well.  Thanks to @beniwohli for reporting.

**Comment by graphaelli** [comment]
fixed on 6.x branch

**Comment by graphaelli** [comment]
once it's merged, fix for elastic/beats#6051 will fix this

**Comment by graphaelli** [comment]
fixed by #670
